src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -179,17 +179,17 @@ export function InternalDashboardTopNav({
         text:
           viewMode === ViewMode.EDIT ? (
             <>
-              {dashboardTitle} <EuiIcon size="s" type="pencil" />
+              {dashboardTitle}
+              <EuiIcon
+                size="s"
+                type="pencil"
+                className="dshTitleBreadcrumbs__updateIcon"
+                onClick={() => dashboard.showSettings()}
+              />
```

Comment:
@cqliu1 @andreadelrio How do we feel about this change? The last breadcrumb is **enforced** to be unclickable, and so the `onClick` is literally being stripped out when building the top nav (see https://github.com/elastic/kibana/pull/166785). This seems like a decent workaround to me (just making the **icon** clickable) but I'm interested in ya'lls thoughts 🤔

Reply:
I'm fine with having just the pencil icon be clickable. That's how I've seen other apps handle it

Reply:
Sounds good to me as well!